cpp/include/cudf/column/dlpack.hpp

```diff
@@ -33,19 +35,20 @@ namespace cudf {
  * 
  * @return Columns converted from DLPack
  */
-std::vector<std::unique_ptr<column>> from_dlpack(
-    DLManagedTensor const& managed_tensor,
+std::unique_ptr<experimental::table> from_dlpack(
+    DLManagedTensor const* managed_tensor,
     rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource());
 
 /**
  * @brief Convert cudf::columns into a DLPack DLTensor
  * 
  * @throw TODO
  * 
- * @param columns Columns to convert to DLPack
+ * @param input Table to convert to DLPack
  * 
  * @return 1D or 2D DLPack tensor (single or multiple columns)
  */
-DLManagedTensor to_dlpack(std::vector<column_view> const& columns);
+DLManagedTensor* to_dlpack(table_view const& input,
```

Comment:
Should this be a unique_ptr?

Reply:
It could be a unique_ptr with a custom deleter that calls the `deleter` function in `DLManagedTensor`, so it would really be unique_ptr<DLManagedTensor, SomeFunctor>. Would that be ok?

Reply:
I guess so. Up to you.

It's not elegant, but that's not your fault. I wish they could have just used C++ for DLpack instead of C. :)

Reply:
I'll leave it with raw pointers for now.
For convenience, I added a unique_ptr wrapper in the unit tests. I could make this the actual return type if we change our mind.
```
struct dlpack_deleter {
  void operator()(DLManagedTensor* tensor) { tensor->deleter(tensor); }
};

using unique_managed_tensor = std::unique_ptr<DLManagedTensor, dlpack_deleter>;
```